### DISCUSSION TITLE
Renew educational discount coupon for GitHub Pro WITHOUT signing up for GitHub Education

### DISCUSSION THREAD

**Discussion by PeterDrake** [community/community#128773] [GitHub Education]
### Hi everyone,

I got an email from you that my educational coupon for GitHub Pro had expired. (I am a professor.) It suggested I go to education.github.com to renew the coupon, but there's nothing on that site about renewing the coupon -- just opportunities to signup for GitHub Education, which I don't want. How do I renew?

**Comment by samus-aran** [comment]
Hi @PeterDrake 

It would be the same way of applying in previous applications via education.github.com (which is currently updating it's website). You use your existing GitHub account to sign in and apply. Here is the direct link - https://education.github.com/discount_requests/application
